ssz/sedes/list.py

```diff
@@ -104,7 +103,8 @@ def _deserialize_stream(self, stream: IO[bytes]) -> Iterable[TDeserialized]:
                     f"element size. data length: {len(data)}  element size: "
                     f"{element_size}"
                 )
-            for segment in partition(element_size, data):
+            for start_idx in range(0, len(data), element_size):
+                segment = data[start_idx: start_idx + element_size]
```

Comment:
I assume that `partition ` is faster since it's in Cython. If so, can we do:
```python
            for segment in partition(element_size, data):
                segment = list(segment)
```
? /cc @pipermerriam

Reply:
I think this is fine for now.  We can attack performance issues once we're a little further along and we know it's a bottleneck.

Reply:
that should be a wrapper in `bytes` then